src/proper_statem.erl

```diff
@@ -308,9 +308,12 @@ commands(Mod) ->
 	 ?SUCHTHAT(
 	    Cmds,
 	    ?LET(List,
-		 ?SIZED(Size,
-			proper_types:noshrink(
-			  commands(Size, Mod, InitialState, 1))),
+                 ?LET(CommandsSizeFactor,
+                      proper_types:parameter('$commands_size_factor', 1),
```

Comment:
I am wondering about the purpose of the `?LET` here...
Long lines aside, wouldn't it be simpler to skip the `?LET` and write the above as
```erlang
             ?SIZED(Size,
￼                   proper_types:noshrink(
￼                       commands(Size * proper_types:parameter('$commands_size_factor', 1),
￼                                Mod, InitialState, 1)))),
```
instead?

(Similar change in the code below, of course.)

Reply:
You're right. I've pushed the changes.